app-config.yaml

```diff
@@ -0,0 +1,31 @@
+app:
+  title: IM GitHub Deployments
+  baseUrl: http://localhost:3000
+  support:
+    url: https://github.com/im-open/im-github-deployments# Used by common ErrorPage
+
+organization:
+  name: im-open
+
+backend:
+  # Used for enabling authentication, secret is shared by all backend plugins
+  # See https://backstage.io/docs/tutorials/backend-to-backend-auth for
+  # information on the format
+  # auth:
+  #   keys:
+  #     - secret: ${BACKEND_SECRET}
+  baseUrl: http://localhost:7007
+  listen:
+    port: 7007
+    # Uncomment the following host directive to bind to all IPv4 interfaces and
+    # not just the baseUrl hostname.
+    # host: 0.0.0.0
+  csp:
+    frame-src: ['miro.com', 'web.powerva.microsoft.com']
```

Comment:
This app-config.yaml is just for running the plugin "stand alone", so it shouldn't have these unneeded CSP frame sources.